README.md

```diff
@@ -196,7 +197,7 @@ Python plugins developers must ensure their plugin to work with all Python versi
  - [Kotlin plugin guideline and example][kotlin-example] by @vincenzopalazzo
 
 [autopilot]: https://github.com/lightningd/plugins/tree/master/archived/autopilot
-[backup]: https://github.com/lightningd/plugins/tree/master/archived/backup
+[backup]: https://github.com/lightningd/plugins/tree/master/backup
```

Comment:
Add this to PR comment

Reply:
Obsolete as this change also is already in `master`.